nlmod/mfoutput/mfoutput.py

```diff
@@ -88,6 +88,12 @@ def _create_da(arr, modelgrid, times):
         flopy modelgrid object
     times : list or array
         list or array containing times as floats (usually in days)
+    add_nans : bool, optional
```

Comment:
Nothing major, but maybe `dry_nans = True` is a bit clearer, instead of add_nans.